flow/scripts/global_route.tcl

```diff
@@ -27,6 +27,18 @@ global_route -guide_file $env(RESULTS_DIR)/route.guide \
                {-congestion_iterations 30 -congestion_report_iter_step 5 -verbose}}]
 
 
+set_placement_padding -global \
+    -left $::env(CELL_PAD_IN_SITES_DETAIL_PLACEMENT) \
+    -right $::env(CELL_PAD_IN_SITES_DETAIL_PLACEMENT)
+repair_antennas -iterations 5
+check_placement -verbose
+
+# Reset padding to zero, as it interfers with recover power and generate
+# overlaps
+set_placement_padding -global \
+    -left 0 \
+    -right 0
+
```

Comment:
No - that seems odd. recover_power doesn't do any placement afaik so I don't see why it would matter. What sort of overlaps?

Reply:
It sounds like a bug as it is not supposed to move anything.  Please look into it.

Reply:
I've found that `replaceCell` function calls opendp when using groute parasitics:
<img width="625" alt="image" src="https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/43450810/15d9fcf0-ab61-4af4-80ea-8bdf1fa88578">

I believe `legalCellPos` tries to legalize the cell if it is out of the die, but since the cell is already in a valid place, the position is not changing. However, it adds the `padLeft` to this position, resulting in the overlaps.
<img width="601" alt="image" src="https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/43450810/706e2c7e-eb79-4c7b-a4af-161035299291">

I will add a check to avoid running the remaining function when the legalized position is equal to the new position, but we should revisit this function in the future.